src/grids/MasonryGrid.ts

```diff
@@ -166,13 +170,20 @@ export class MasonryGrid extends Grid<MasonryGridOptions> {
       gap,
       align,
     } = this.options;
+    const containerInlineSize = this.getContainerInlineSize();
     const columnSizeOption = this.columnSize || this.outlineSize;
-    const column = this.column || this.outlineLength || 1;
+    const columnOption = this.column || this.outlineLength;
+    let column = columnOption || 1;
```

Comment:
I wonder is it possible left-hand side of these values to be `0`?
I think in that case it would be better to use `??` instead.

Reply:
If the value is not set, it will be 0.

Reply:
I mean, in that case shouldn't it be like `this.columnSize ?? this.outlineSize` instead?
As `outlineSize` will be used instead if `columnSize` is `0` when using `||`

Reply:
The priority is columnSize > outlineSize, column > outlineLength.

Both columnSize and outlineSize are integers, not null or undefined.